### ISSUE TITLE
"image return" option activation freezes the main screen current improvisation countdown 

### ISSUE THREAD

**Issue by fitchMitch** [Ealhad/kazoo#18]
**"_Retour écran_" :** 
Whenever activated, countdowns of both match and improvisation on main screen freeze. When the option get deactivated, countdown starts again. It should always count down whatever the chosen option.

**Comment by BuonOmo** [comment]
I agree with you, however, I cannot reproduce the issue on my computer... Could tell me precise steps. Or run `command` + `ctrl` + `i` to open your console and tell me if there is any error thrown?

**Comment by fitchMitch** [comment]
Consoles of both screen are empty. 



As you can see, the content of both countdown displays are equal, but when I switch screens to follow the countdown on the main official one, countdown value is  freeezed to a close value from the one I just saw on the paramaters screen. Switching tab again to the parameters screen I can see, that the countdown just goes on normally. Switching tab again to the main screen , I can see the value has changed to the latest value but has freezed again. 
However, when switching "Retour écran" to off, the behaviour is not the same. I can switch tabs and follow countdown time decrease smoothly on each of the tabs.
Hope it helps to reproduce. 

Other: Firefox 64 / Ubuntu 18

![buonomo](https://user-images.githubusercontent.com/2885396/50078993-b7d8f400-01e8-11e9-99f1-f323ea4da7dd.png).
